### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by Mainagithubrit**
storage_get_count

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # This file lists all individuals having contributed content to the repository.
 
 
-Jennifer Huang <133@holbertonschool.com>
-Alexa Orrico <210@holbertonschool.com>
-Joann Vuong <130@holbertonschool.com> 
+Francis Njoroge <njorogefrancismaina@gmail.com>
```
Comment (Mainagithubrit) [inline AUTHORS:4]:
```suggestion
Francis Njoroge <njorogefrancismaina@gmail.com>
```

File: AUTHORS [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
+Emmanuel Ejeagha <nuel6700@gmail.com>
```
Comment (Mainagithubrit) [inline AUTHORS:5]:
```suggestion
Emmanuel Ejeagha <nuel6700@gmail.com> 
```

File: api/v1/app.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
```
Comment (Mainagithubrit) [inline api/v1/app.py:1]:
```suggestion
#!/usr/bin/python3
```

File: api/v1/app.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+"""Creating a Flask app"""
```
Comment (Mainagithubrit) [inline api/v1/app.py:2]:
```suggestion
"""Creating a Flask app"""
```

File: api/v1/app.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (Mainagithubrit) [inline api/v1/app.py:3]:
```suggestion

```

File: api/v1/app.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+from os import getenv
```
Comment (Mainagithubrit) [inline api/v1/app.py:4]:
```suggestion
from os import getenv
```

File: README.md [hunk 7]
```diff
@@ -154,8 +154,8 @@ EOF  all  create  destroy  help  quit  show  update
 No known bugs at this time. 
 
 ## Authors
-Alexa Orrico - [Github](https://github.com/alexaorrico) / [Twitter](https://twitter.com/alexa_orrico)  
-Jennifer Huang - [Github](https://github.com/jhuang10123) / [Twitter](https://twitter.com/earthtojhuang)
+Emmanuel Ejeagha - [Github](https://github.com/Emmanuel-Ejeagha) / [Twitter](https://twitter.com/emma_ejeagha)
```
Comment (Mainagithubrit) [inline README.md:157]:
```suggestion
Emmanuel Ejeagha - [Github](https://github.com/Emmanuel-Ejeagha) / [Twitter](https://twitter.com/emma_ejeagha)
```

File: README.md [hunk 8]
```diff
@@ -154,8 +154,8 @@ EOF  all  create  destroy  help  quit  show  update
...
+Francis Njoroge - [Github](https://github.com/Mainagithubrit)
```
Comment (Mainagithubrit) [inline README.md:158]:
```suggestion
Francis Njoroge - [Github](https://github.com/Mainagithubrit) 
```

File: api/v1/app.py [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+from models import storage
```
Comment (Mainagithubrit) [inline api/v1/app.py:5]:
```suggestion
from models import storage
```

**Comment by Mainagithubrit** [comment]
storage

**Comment by Mainagithubrit** [comment]
str

**Comment by Emmanuel-Ejeagha** [comment]
Done

**Review by Emmanuel-Ejeagha — APPROVED** [review]
Approved a request

**Pull request closed without merging**